autocommit.js

```diff
@@ -33,8 +34,14 @@ try {
   process.exit(1);
 }
 
-if (!config.openAiKey) {
-  console.error("Please set OPENAI_API_KEY");
+if (!config.openAiKey && !config.azureOpenAiKey) {
+  console.error("Please set OPENAI_API_KEY or AZURE_OPENAI_API_KEY");
+  process.exit(1);
+}
+// if any settings related to AZURE are set, if there are items that are not set, will error.
```

Comment:
new line before comment
```suggestion

// if any settings related to AZURE are set, if there are items that are not set, will error.
```